aptos-move/framework/aptos-stdlib/sources/pool_u64_unbound.spec.move

```diff
@@ -117,54 +121,60 @@ spec aptos_std::pool_u64_unbound {
         }
     }
 
-    spec multiply_then_divide(_pool: &Pool, x: u128, y: u128, z: u128): u128 {
+    spec multiply_then_divide(_self: &Pool, x: u128, y: u128, z: u128): u128 {
```

Comment:
Does dot notation allow a parameter named `_self`?

Reply:
I don't think so but adding `self` will lead to a compiler warning. Not sure what to do here.

Reply:
cc @wrwg

Reply:
Oops. We need to turn this warning off, so one is able to provide an unused self parameter. If this is only for those spec functions, the `_self` is fine for now, but I opened https://github.com/aptos-labs/aptos-core/issues/14361 for this.

Reply:
@wrwg  Both `pool_u64` and `pool_u64_unbound` smart contract code define the function `multiply_then_divide`.  In this case, should we still keep `_self`?